src/frontend/components/search/Search.test.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { shallow, ShallowWrapper } from 'enzyme'
+import Search from './Search'
+
+describe('Search tests', () => {
+  let wrapper: ShallowWrapper
+
+  beforeEach(() => {
+    wrapper = shallow(<Search />)
+  })
+
+  it('button is disabled when input is empty', () => {
+    const btn = wrapper.find('button')
+    expect(btn.prop('disabled')).toBeTruthy()
```

Comment:
I would avoid `toBeTruthy()`. If the value is supposed to be a non-empty string, then I would test for that. Makes your test more specific and likely to fail if it changes to an unexpected value

Unless the `prop()` function could return multiple data types, then `toBeTruthy()` might be ok